src/endpoints/kp_endpoints.py

```diff
@@ -15,7 +15,7 @@ def fetch_kp_token():
     return jsonify(kp_client.fetch_token())
 
 
-@api_kp_bp.route('/search/person', methods=['POST'])
+@api_kp_bp.route('/search/person', methods=['POST'])  # TODO: Could this not be changed to a GET with params? - maybe name changed to : /person/search
```

Comment:
You can just remove this TODO

Reply:
I forgot to remove it, but I think it's fine. We should change this at some point.